ding_dibs.module

```diff
@@ -386,6 +386,28 @@ function ding_dibs_menu() {
     'file' => 'ding_dibs.reports.inc',
   );
 
+  $items['admin/config/payment/ding_dibs/emails'] = array(
+    'title' => 'Ding DIBS setting emails',
+    'description' => 'Ding DIBS setting emails.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('ding_dibs_setting_emails'),
+    'access arguments' => array('administer dibs settings'),
+    'type' => MENU_LOCAL_TASK,
+    'weight' => 5,
+    'file' => 'ding_dibs.email_reports.inc',
+  );
+
+  $items['admin/config/payment/ding_dibs/cron_report'] = array(
```

Comment:
It seems like this would be a better fit under Drupals Report section e.g. `admin/reports/ding_dibs`?